store/tikv/prewrite.go

```diff
@@ -140,6 +151,27 @@ func (action actionPrewrite) handleSingleBatch(c *twoPhaseCommitter, bo *Backoff
 					c.run(c, nil)
```

Comment:
Don't run it in 1pc.

Reply:
Actually this won't happen because 1PC txn only has one batch so the transaction must be quite small. Anyway, it would be better to add a check again.